yt_dlp/extractor/drooble.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    try_get,
+)
+
+
+class DroobleIE(InfoExtractor):
+    _VALID_URL = r'https?://drooble\.com/(?:(?P<user>[^/]+)?/?(?P<kind>song|videos|music/albums)/(?P<id>\d+)|(?P<id_2>[^/]+)/(?P<kind_2>videos|music))'
+    _TESTS = [{
+        'url': 'https://drooble.com/song/2858030',
+        'md5': '5ffda90f61c7c318dc0c3df4179eb064',
+        'info_dict': {
+            'id': '2858030',
+            'ext': 'unknown_video',
```

Comment:
it shouldn't be this lol
U need to manually set ext during extraction ig

Reply:
it is acceptable only if the site uses multiple formats and there is no way to tell them apart